prac_07/myguitars.py

```diff
@@ -30,4 +37,21 @@ def load_guitars(filename):
     return guitars
 
 
+def get_new_guitar():
+    name = input("Name: ")
+    year = int(input("Year: "))
+    cost = float(input("Cost: $"))
+    guitar = Guitar(name, year, cost)
```

Comment:
You could return this whole line instead of setting a variable and returning that.